src/main/resources/static/js/draggable-utils.js

```diff
@@ -8,71 +8,118 @@ const DraggableUtils = {
-
-            // check if control key is pressed
-            if (event.ctrlKey) {
-              const aspectRatio = target.offsetWidth / target.offsetHeight;
-              // preserve aspect ratio
-              let width = event.rect.width;
-              let height = event.rect.height;
-
-              if (Math.abs(event.deltaRect.width) >= Math.abs(event.deltaRect.height)) {
-                height = width / aspectRatio;
-              } else {
-                width = height * aspectRatio;
-              }
-
-              event.rect.width = width;
-              event.rect.height = height;
+      },
+    })
+    .resizable({
+      edges: { left: true, right: true, bottom: true, top: true },
+      listeners: {
+        move: (event) => {
+          var target = event.target;
+          var x = parseFloat(target.getAttribute("data-bs-x")) || 0;
+          var y = parseFloat(target.getAttribute("data-bs-y")) || 0;
+
+          // check if control key is pressed
+          if (event.ctrlKey) {
+            const aspectRatio = target.offsetWidth / target.offsetHeight;
+            // preserve aspect ratio
+            let width = event.rect.width;
+            let height = event.rect.height;
+
+            if (Math.abs(event.deltaRect.width) >= Math.abs(
+                event.deltaRect.height)) {
+              height = width / aspectRatio;
+            } else {
+              width = height * aspectRatio;
             }
 
-            target.style.width = event.rect.width + "px";
-            target.style.height = event.rect.height + "px";
+            event.rect.width = width;
+            event.rect.height = height;
+          }
 
-            // translate when resizing from top or left edges
-            x += event.deltaRect.left;
-            y += event.deltaRect.top;
+          target.style.width = event.rect.width + "px";
+          target.style.height = event.rect.height + "px";
 
-            target.style.transform = "translate(" + x + "px," + y + "px)";
+          // translate when resizing from top or left edges
+          x += event.deltaRect.left;
+          y += event.deltaRect.top;
 
-            target.setAttribute("data-bs-x", x);
-            target.setAttribute("data-bs-y", y);
-            target.textContent = Math.round(event.rect.width) + "\u00D7" + Math.round(event.rect.height);
+          target.style.transform = "translate(" + x + "px," + y + "px)";
 
-            this.onInteraction(target);
-          },
+          target.setAttribute("data-bs-x", x);
+          target.setAttribute("data-bs-y", y);
+          target.textContent = Math.round(event.rect.width) + "\u00D7"
+              + Math.round(event.rect.height);
+
+          this.onInteraction(target);
         },
+      },
+
+      modifiers: [
+        interact.modifiers.restrictSize({
+          min: {width: 5, height: 5},
+        }),
+      ],
+      inertia: true,
+    });
+    if(window.location.pathname.endsWith('sign')) {
```

Comment:
TODO: extend this to other features.

Reply:
Could any other features use this functionality? If not, we should move it into it's own thing.

Reply:
add-image could support this

Reply:
Yes, it already works for add image if the if statement is left out, but if two images are added, both are moving. I will try and figure out a way so only one is moving.

Reply:
I extended this feature to add-Image. Arrow key support is enabled for an element as soon as it is added. By clicking or dragging a different element arrow key support is enabled for that element.